app/services/foreman_wreckingball/vmware_cluster_importer.rb

```diff
@@ -20,9 +20,9 @@ def import!
     def delete_removed_clusters
       delete_query = ::ForemanWreckingball::VmwareCluster.where(:compute_resource => compute_resource).where.not(:name => cluster_names)
       counters[:deleted] = if ActiveRecord::Base.connection.adapter_name.downcase.starts_with? 'mysql'
-                             ::ForemanWreckingball::VmwareCluster.where(:id => delete_query.pluck(:id)).delete_all
```

Comment:
`delete_all` was used here to generate a single `DELETE FROM` sql query. I don't think we need to distinguish between mysql and other DBMS because `destroy_all` will use a query per record.

Reply:
I would remove the if condition and just call `delete_query.destroy_all` in every case.

Reply:
AH I see! Thanks